geonode/base/enumerations.py

```diff
@@ -503,3 +504,5 @@
 )
 
 LAYER_TYPES = ['vector', 'raster', 'remote', 'vector_time']
+
+DEFAULT_MAX_BEFORE_UPLOAD_SIZE = (settings.DEFAULT_MAX_UPLOAD_SIZE * 2) + 2097152
```

Comment:
@mattiagiupponi we don't want to use an enumeration. The size for the upload handler must be calculated according to this formula but dynamically from the dataset (or document) upload size set inside the DB.